WireDomain/Sources/WireDomain/Repositories/TeamRepository.swift

```diff
@@ -44,6 +44,17 @@ protocol TeamRepositoryProtocol {
 
     func fetchSelfLegalholdStatus() async throws -> LegalholdStatus
 
+    /// Deletes the member of a team.
+    /// - Parameter time: The time the member left the team.
+    /// - Parameter userID: The ID of the team member.
+    /// - Parameter teamID: The ID of the team.
+
+    func deleteTeamMember(
+        time: Date,
+        userID: UUID,
+        teamID: UUID
+    ) async throws
```

Comment:
nitpick: I think a more idiomatic swift signature would be something like:

```swift
func deleteMembership(
    forUser userID: UUID, 
    fromTeam teamID: 
    at time: Date
) async throws
```

Reply:
fixed thx